easy-ui-react/src/EmptyStateCard/EmptyStateCard.tsx

```diff
@@ -1,168 +1,163 @@
-import React, { ReactNode, useMemo } from "react";
+import React, { ReactNode } from "react";
 import { Text, TextProps } from "../Text";
 import { Card } from "../Card";
-import { VerticalStack } from "../VerticalStack";
-import {
-  flattenChildren,
-  getDisplayNameFromReactNode,
-} from "../utilities/react";
-import { ResponsiveSpaceScale } from "../types";
+import { VerticalStack, VerticalStackProps } from "../VerticalStack";
+import { HorizontalStack, HorizontalStackProps } from "../HorizontalStack";
+import { classNames } from "../utilities/css";
+
+import { BASE_64_BLUE_BOX, BASE_64_CLAPPING_BENJAMIN } from "./utilities";
```

Comment:
having consumer pass in the asset seemed odd for a component like this, but also this resulted in a big base 64 encoding for the clapping benjamin, probably need a better way to handle this in the future